.travis.yml

```diff
@@ -16,41 +16,25 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7
+    - php: 7.3
       env:
         - DEPS=latest
-    - php: 7.1
-      env:
-        - DEPS=lowest
-    - php: 7.1
+    - php: 7.4
       env:
         - DEPS=latest
         - CS_CHECK=true
         - TEST_COVERAGE=true
-    - php: 7.2
+    - php: nightly
```

Comment:
Could you change the travis configuration as in laminas/laminas-servicemanager#64?
Travis added PHP 8.0 [yesterday](https://travis-ci.community/t/php-8-0-missing/10132/23) support for the matrix and thus we can add `8.0` aswell as `nightly` where nightly is allowed to fail.

There is an example in the linked PR 👍

Reply:
Cool, I've added the PHP 8 specific tests in Travis, and allowed `nightly` to fail.

I've had a bit of a look at the PHP Curl library and can't find much info about what constitutes good performance. An easy way to fix the test would be to change the timeout that it's tested against from 20 to 25 or 30, hmm.